index.js

```diff
@@ -0,0 +1,110 @@
+const addButtonElement = document.querySelector('.add');
+const inputElement = document.querySelector('#task');
+const toDo = document.querySelector('.to-do');
+const doing = document.querySelector('.doing');
+const done = document.querySelector('.done');
+
+let deleteButton;
```

Comment:
Having this many global variables declared at the very top is often a sign of bad design.
In your code, it is not obviously apparent where the `deleteButton` and other variables are declared.

My suggestion is to modify the `createButtonArea` function so that it returns the following:
```
return {
  deleteButton,
  deleteButtonDoingSection,
  deleteButtonDoneSection,
  rightArrowButton,
  rightArrowButtonDoingSection,
  leftArrowButtonDoingSection,
  leftArrowButtonDoneSection
}
```
Thanks to the above approach, those variables no longer need to be global.